### PR TITLE
Align user-guide: clone with a snapshot source is supported

### PR DESCRIPTION
**Pull request by iholder101**
Align user-guide: clone with a snapshot source is supported.

This supported was added as part of the following PR: https://github.com/kubevirt/kubevirt/pull/8458

### CODE REVIEW

File: docs/operations/clone_api.md [hunk 1]
```diff
@@ -139,14 +140,13 @@ kubectl get vm vm-clone-target -o yaml
...
-* Add more supported source types. The most important is `VirtualMachineSnapshot` (see section below for more info). We
-  also intend to support `VirtualMachineInstace` types in the future.
+* Add more supported source types, like `VirtualMachineInstace` in the future.
 * Add a cross-namespace clone support. This needs to be supported for snapshots / restores first.
 
 ### Using clones as a "golden VM image"
 
-One of the great things that could be accomplished with the clone API when `VirtualMachineSnapshot`s are supported
-as source types is to create "golden VM images" (a.k.a. Templates / Bookmark VMs / etc). In other words, the following
+One of the great things that could be accomplished with the clone API when the source if of kind `VirtualMachineSnapshot`
```
Comment (jean-edouard) [inline docs/operations/clone_api.md:148]: `the source if of kind`? Seems like a typo :)

**Review by jean-edouard — COMMENTED** [review]
Looks good, just one potential typo.
Any chance you could add a link to the commit that added support for cloning a snapshot to the PR desc? Thanks!
/approve
/hold

File: docs/operations/clone_api.md [hunk 2]
```diff
@@ -139,14 +140,13 @@ kubectl get vm vm-clone-target -o yaml
 The clone API is in an early alpha version and may change dramatically. There are many improvements and features
 that are expected to be added, the most significant goals are:
 
-* Add more supported source types. The most important is `VirtualMachineSnapshot` (see section below for more info). We
-  also intend to support `VirtualMachineInstace` types in the future.
+* Add more supported source types, like `VirtualMachineInstace` in the future.
```
Comment (cwilkers) [inline docs/operations/clone_api.md:143]:
```suggestion
* Add more supported source types like `VirtualMachineInstace` in the future.
```
Don't need a comma here

File: docs/operations/clone_api.md (hunk 1 above)
Comment (iholder101) [reply docs/operations/clone_api.md]: Yup! Indeed a typo :)
Fixed to `the source is of kind`

File: docs/operations/clone_api.md (hunk 2 above)
Comment (iholder101) [reply docs/operations/clone_api.md]: Done, thanks!

**Comment by iholder101** [comment]
> Looks good, just one potential typo. Any chance you could add a link to the commit that added support for cloning a snapshot to the PR desc? Thanks! /approve /hold

Great idea! Added a link to the PR in the description.
Thanks!

/unhold

**Comment by Barakmor1** [comment]
/lgtm 
Thanks!

**Pull request merged**